projects/cds/src/cms-components/profile-tag/profile-tag.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface ProfileTagWindowObject {
+  Y_TRACKING: {
+    push?: Function;
+    q?: any[][];
+  };
+}
+
+export interface ProfileTagJsConfig {
+  tenant: string;
+  siteId: string;
+  spa: boolean;
+  javascriptUrl: string;
+  configUrl: string;
+  allowInsecureCookies?: boolean;
+  gtmId?: string;
+  profileTagEventReciever: Function;
```

Comment:
This object is sent to our 3rd party, 'profiletag.js' script, so I guess I don't see a reason that it would be serialized or placed in the ngrx store.